spec/models/prison_spec.rb

```diff
@@ -2,21 +2,59 @@
 
 RSpec.describe Prison, type: :model do
   describe '#offenders' do
-    let(:offenders) { described_class.new("LEI").offenders }
+    subject { described_class.new("LEI").offenders }
 
     it "get first page of offenders for a specific prison",
        vcr: { cassette_name: :offender_service_offenders_by_prison_first_page_spec } do
-      offender_array = offenders.first(9)
+      offender_array = subject.first(9)
       expect(offender_array).to be_kind_of(Array)
       expect(offender_array.length).to eq(9)
       expect(offender_array.first).to be_kind_of(Nomis::OffenderSummary)
     end
 
     it "get last page of offenders for a specific prison", vcr: { cassette_name: :offender_service_offenders_by_prison_last_page_spec } do
-      offender_array = offenders.to_a
+      offender_array = subject.to_a
       expect(offender_array).to be_kind_of(Array)
       expect(offender_array.length).to be > 800
       expect(offender_array.first).to be_kind_of(Nomis::OffenderSummary)
     end
+
+    context 'when there are exactly 200 offenders' do
+      let(:offenders) { build_list(:nomis_offender, 200) }
+
+      before do
+        stub_auth_token
+        stub_request(:get, "#{ApiHelper::T3}/locations/description/LEI/inmates?convictedStatus=Convicted&returnCategory=true").
+          with(
+            headers: {
+              'Page-Limit' => '1',
+              'Page-Offset' => '1',
+            }).
+          to_return(
+            body: {}.to_json,
+            headers: { 'Total-Records' => '200' }
+          )
```

Comment:
Looks like we might not need this stub?

We don't do a 'Page-Limit' => '1', 'Page-Offset' => '1' request any more.